judge/views/organization.py

```diff
@@ -401,43 +402,53 @@ class ProblemListOrganization(CustomOrganizationMixin, ProblemList):
     context_object_name = 'problems'
     template_name = 'organization/problem-list.html'
 
-    def get_title(self):
-        return _('Problems list of %s') % self.organization.name
-
-    def get_content_title(self):
-        return format_html(_('Problems list of') + ' <a href="{1}">{0}</a>', self.organization.name,
-                           self.organization.get_absolute_url())
+    def get_hot_problems(self):
+        return None
 
     def get_context_data(self, **kwargs):
         context = super(ProblemListOrganization, self).get_context_data(**kwargs)
         context['organization'] = self.organization
+        context['title'] = self.organization.name
```

Comment:
I think we can make use of the `get_title` & `get_content_title`.

Perhaps override those functions in `CustomOrganizationMixin`?

Reply:
I don't think it's a good idea. `ProblemCreateOrganization` and `ContestCreateOrganization`, which need custom titles, also use `CustomOrganizationMixin`.